drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -1897,6 +1906,36 @@ bool RasterizerStorageGLES2::material_casts_shadows(RID p_material) {
 	return casts_shadows;
 }
 
+bool RasterizerStorageGLES2::material_uses_tangents(RID p_material) {
+	Material *material = material_owner.get(p_material);
+	ERR_FAIL_COND_V(!material, false);
+
+	if (!material->shader) {
+		return false;
+	}
+
+	if (material->shader->dirty_list.in_list()) {
+		_update_shader(material->shader);
+	}
```

Comment:
I've added a shader update here and for ensure_correct_normals as well because information would be updated only on the next render update otherwise, and it wouldn't allow MeshInstance to check materials only once on init or when they are modified.

It didn't seem to cause any issue in the tests I've made.